.github/workflows/test-win-exe-w-embed-py.yaml

```
@@ -1,203 +1,208 @@
 name: Test streamlit executable for Windows with embeddable python
-on: 
-  push:
+on:
+   pull_request:
     branches: [ "main" ]
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)